font/src/ft/mod.rs

```diff
@@ -109,7 +109,7 @@ impl Rasterize for FreeTypeRasterizer {
         // Fallback for bitmap fonts which do not provide underline metrics
         if underline_position == 0. {
             underline_thickness = (descent / 5.).round();
-            underline_position = descent / 2.;
+            underline_position = descent * 2.;
```

Comment:
This doesn't make much sense as a metric. It will probably just always put the underline at the far bottom of the glyph, which likely looks really strange for most fonts.

The reasoning behind `descent / 2` is that the descent is the distance from the bottom of the glyph to the bottom of the cell, so half of that should put it about in the middle of the space below the glyph.

If anything, I'd probably do something more like `underline_position = descent / 2. - (underline_thickness / 2.).ceil()`.